covid_model/model_fit.py

```diff
@@ -7,15 +7,15 @@
 
 import scipy.optimize as spo
 
-from data_imports import ExternalHosps, ExternalData
-from model import CovidModel
-from model_specs import CovidModelSpecifications
+from covid_model.data_imports import ExternalHosps, ExternalData
```

Comment:
this avoids having covid_model in the PYTHONPATH, right?

Reply:
Yes, I think so. I'd fixed it everywhere else already, but if you see any others we should update everything to use `covid_model.`